leapp/libraries/stdlib/path.py

```diff
@@ -0,0 +1,60 @@
+import os
+
+
+def get_file_path(directories, name):
+    """
+    Finds the first matching file path within directories.
+
+    :param name: Name of the file
+    :type name: str
+    :return: Found file path
+    :rtype: str or None
+    """
+    for path in directories:
+        path = os.path.join(path, name)
+        if os.path.isfile(path):
+            return path
+
+
+def get_folder_path(directories, name):
+    """
+    Finds the first matching folder path within directories.
+
+    :param name: Name of the folder
+    :type name: str
+    :return: Found folder path
+    :rtype: str or None
+    """
+    for path in directories:
+        path = os.path.join(path, name)
+        if os.path.isdir(path):
+            return path
+
+
+def get_tool_path(directories, name):
+    """
+    Finds the first matching executable file within directories.
+
+    :param name: Name of the file
+    :type name: str
+    :return: Found file path
+    :rtype: str or None
+    """
+    for path in directories:
+        path = os.path.join(path, name)
+        if os.path.isfile(path) and os.access(path, os.X_OK):
+            return path
+
+
+# NOTE(ivasilev) You can use this standalone with
+# manager = load_repositories_from('repo_path', '/etc/leapp/repo.d/', manager=None)
+def get_common_file_path(manager, name):
```

Comment:
`manager` is implementing the repository protocol, in terms of Python 3.6+ typing.Protocol 
So I would call it `repository`, not `manager`

An instance of `leapp.repository.Manager` is returned, when more than one repository has been scanned

Reply:
Should be done now for all 3 functions